.eslintrc

```diff
@@ -14,10 +16,16 @@
     "sourceType": "module"
   },
   "plugins": ["react"],
-  "rules": {},
+  "rules": {
+    "import/order": ["error", { "newlines-between": "always" }]
+  },
   "settings": {
     "react": {
       "version": "detect"
-    }
+    },
+    // ESLint doesn't find React Native components
+    // Remove this setting when this issue is fixed.
+    // https://github.com/facebook/react-native/issues/28549
+    "import/ignore": ["react-native"]
```

Comment:
can you please create an issue in this repo which links that issue? otherwise we'll forget. good catch though

Reply:
Done https://github.com/nearform/optic-expo/issues/14